### ISSUE TITLE
**CM-11** apps are being instantly killed

### ISSUE THREAD

**Issue by yglo** [LegacyXperia/local_manifests#1078]
Hi. What is going on with the latest release of **cm-11** on xperia neo v(haida)(**cm-11-20160417-UNOFFICIAL-LegacyXperia-haida** )?... I'm experiencing an instant kill of every app that had been switched to the background. It had never happened before. Well, at least in previous release that i came from, which was **cm-11-20150501-UNOFFICIAL-LegacyXperia-haida**... And no, I've not changed '_Background process limit'_ to _no background processes._

**Comment by koxelez** [comment]
Make sure none of your apps is gobbling your RAM. I've hand my Mini Pro do this quite often but mostly happens many hours after boot. I've always assumed it was related to a memory leak.

EDIT: you could always catch a logcat and see what the OS is doing

**Comment by yglo** [comment]
Hello. Thanks for the reply. I dont know if any app indeed is gobbling my RAM memory but i assume that none is. It is just flashed brand new ROM and i've restored a lot less apps that i've had before. So i actually think it should run even smoother than the old messy system. Now i'm kinda disappointed and dont know what to do. Is there any way to figure it out if it's ROM's fault? How to get this _logcat_ and handle this?

**Comment by dev-tee** [comment]
There is a wiki entry about reporting bugs that has a guide on how to collect a logcat among other things.
[LegaxyXperia Wiki - Reporting Problems](https://github.com/LegacyXperia/Wiki/wiki/Reporting-problems)

tl;dr:
`adb logcat -d`
prints the logcat to the screen
